pkg/efi/manager.go

```diff
@@ -57,9 +59,10 @@ func NewBootManagerForVariables(efivars Variables) (BootManager, error) {
 		if err != nil {
 			return BootManager{}, fmt.Errorf("cannot read %s: %v", name, err)
 		}
-		entry.LoadOption, err = efi.ReadLoadOption(bytes.NewReader(entry.Data))
+		entry.LoadOption, err = bm.efivars.ReadLoadOption(bytes.NewReader(entry.Data))
 		if err != nil {
-			return bm, err
+			logger.Warnf("Error reading efi load option for '%s': %s", name, err.Error())
+			continue
```

Comment:
I guess this is the actual fix. Where do we see this issue? Which is the impact of this bug. I am wondering if this is something to be backported somewhere.

Reply:
Correct, this is from the linked issue, which is coming from [this harvester issue](https://github.com/harvester/harvester/issues/4528)